src/mapping.ts

```diff
@@ -54,21 +60,21 @@ export function handleTransferSingle(event: TransferSingle): void {
   toUser.pods = toUser.pods.concat([id.toString()]);
   toUser.save();
 
-  const fromUser = User.load(from.toHex());
+  let fromUser = User.load(from.toHex());
   // The fromUser should never be null, because the User is created
   // when they first receive a token.
   // This check is just assurance.
   if (fromUser !== null) {
     // Remove the pod from the user's "pods" array
-    fromUser.pods = fromUser.pods.splice(fromUser.pods.indexOf(id.toString()), 1);
+    fromUser.pods = removeItem(fromUser.pods, id.toString());
```

Comment:
i think we can replace this with a lodash function
```js
fromUser.pods = _.remove(fromUser.pods, elem => elem === id.toString());
```

Reply:
This code snippet is going to be removed with my next PR anyway so lets just keep it for now